### PR TITLE
feat: Each second without first, add user input, remove spaces #11

### PR DESCRIPTION
**Pull request by ehryhorenko**

### CODE REVIEW

File: entry_tasks/11_each_second/11_each_second.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+if __name__ == '__main__':
+    my_list = [3, 2, 1, 4, 5,  2, 'ss', 'q']
```
Comment (mykytapavlov) [inline entry_tasks/11_each_second/11_each_second.py:2]: Please make this to be input() by user

File: entry_tasks/11_each_second/11_each_second.py [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+    my_list_string = [str(x) for x in my_list]
+    my_list_string = [' ' + x + '' for x in my_list_string]
```
Comment (mykytapavlov) [inline entry_tasks/11_each_second/11_each_second.py:4]: so in user input those spaces would exist, and will have to remove them

**Review by mykytapavlov — APPROVED** [review]

File: entry_tasks/11_each_third/11_each_third.py [hunk 3]
```diff
@@ -0,0 +1,6 @@
+if __name__ == '__main__':
+    input_list = input('List of numbers or letters, separated by comma: ') #3, 2, 1, 4, 5,  2, ss, q
+    user_list = input_list.split(',')
+    user_list = ['' + x + '' for x in user_list]
```
[the remark this answers is not in the document]
Comment (mykytapavlov) [reply entry_tasks/11_each_third/11_each_third.py:4]: and please, squash the commits [useful links](https://github.com/mykytapavlov/python-beginner#git)

[the remark this answers is not in the document]
Comment (ehryhorenko) [reply entry_tasks/11_each_third/11_each_third.py:4]: Maybe I didn't understand the task properly. I mean the output.
In task:
>>> 3, 2, 1, 4, 5,  2, ss, q **_- this is the input_**
>>> [' 1', ' 5', ' ss'] - **_and this is the output, right?_** 

And in output, there are spaces (in quotes before symbol).

[the remark this answers is not in the document]
Comment (mykytapavlov) [reply entry_tasks/11_each_third/11_each_third.py:4]: yes, that's correct, but if you didn't want to remove the spaces, why did you add:
```py
user_list = ['' + x + '' for x in user_list]
```
I mean user_list on line 3 and 4 is the same

[the remark this answers is not in the document]
Comment (mykytapavlov) [reply entry_tasks/11_each_third/11_each_third.py:4]: user_list = ['' + x + '' for x in user_list] - you just add "nothing" before and after each element, got it?

[the remark this answers is not in the document]
Comment (ehryhorenko) [reply entry_tasks/11_each_third/11_each_third.py:4]: You are right! Line N4 is extra and not needed. Thanks.
But before I will push changes, should I remove the extra spaces, so the output will look like
['1', '5', 'ss'] ?

**Review by mykytapavlov — APPROVED** [review]

**Pull request merged**